components/constellation/constellation.rs

```diff
@@ -614,6 +618,13 @@ impl<Message, LTF, STF> Constellation<Message, LTF, STF>
 
         if let Some(child_process) = child_process {
             self.child_processes.push(child_process);
+
+            let script_thread_id = ScriptThreadId::new();
+            self.script_threads.insert(script_thread_id, pipeline.script_chan.clone());
+
+            if let Some(ref debugger_chan) = self.debugger_chan {
+                debugger::script_thread_added(debugger_chan, script_thread_id);
+            }
```

Comment:
Why is this code only run if there's a new process? I don't see why multiprocess makes a difference here. Also, this code is running every time you create a new pipeline, not just when a new script thread is created.